CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## 0.1.12
+
+### Preview features
+
+- Hug multiline-strings in preview style ([#9243](https://github.com/astral-sh/ruff/pull/9243))
+- [\`flake8-bandit\`] Add `ssl-with-no-version` (`S504`) ([#9384](https://github.com/astral-sh/ruff/pull/9384))
+- [\`flake8-bandit\`] Implement `ssl-insecure-version` (`S502`) ([#9390](https://github.com/astral-sh/ruff/pull/9390))
+- [\`flake8-bandit\`] Implement `ssl-with-bad-defaults` (`S503`) ([#9391](https://github.com/astral-sh/ruff/pull/9391))
+- [\`flake8-bandit\`] Implement suspicious import rules (`S4XX`) ([#8831](https://github.com/astral-sh/ruff/pull/8831))
+- [\`flake8-simplify\`] Implement `zip-dict-keys-and-values` (`SIM911`) ([#9460](https://github.com/astral-sh/ruff/pull/9460))
+- [\`pyflakes\`] Add a fix for `redefined-while-unused` (`F811`) ([#9419](https://github.com/astral-sh/ruff/pull/9419))
+- [\`pylint\`] Implement `unnecessary-dunder-call` (`C2801`) ([#9166](https://github.com/astral-sh/ruff/pull/9166))
+- [\`ruff\`] Add `parenthesize-changed-operators` (`RUF021`) to enforce parentheses in `a or b and c` ([#9440](https://github.com/astral-sh/ruff/pull/9440))
+
+### Rule changes
+
+- [\`flake8-boolean-trap\`] Allow Boolean positionals in setters ([#9429](https://github.com/astral-sh/ruff/pull/9429))
+- [\`flake8-bugbear\`] Include subscripts and attributes in static key rule ([#9416](https://github.com/astral-sh/ruff/pull/9416))
+- [\`flake8-bugbear\`] Support variable keys in static dictionary key rule ([#9411](https://github.com/astral-sh/ruff/pull/9411))
```

Comment:
Looks like these are both changes to RUF011 -- is it correct to classify them as flake8-bugbear?

Reply:
Oops, thanks. This rule was moved to flake8-bugbear, but that PR hasn't actually merged yet, my mistake!